### ISSUE TITLE
Wrong rendering of line plots on the Web Interface (missing values)

### ISSUE THREAD

**Issue by habichta** [wandb/client#1374]
**Environment:**
- wandb 0.10.4
- tensorflow 2.1.0
- keras 2.2.4
- Keras WandbCallback
- We are using the metrics as provided by Keras

**Issue:**
We noticed that the line plots on the W&B Web interface behave erratically, sometimes missing certain values. I have attached two pictures to show the problem. In the first picture, one can see the downward spike of val_true_negatives to almost 0 between epoch 1k and 1.5k. And the corresponding upward spike in val_true_positives.

The training continuous and in picture 2 that drop suddenly disappears, giving a wrong impression about the performance of the model. After some epochs, the spikes appear and disappear again. We spotted this problem with any metric. The correct values appear in the W&B logs. So I assume it is a visualization issue. Which, however, is quite serious for us, since we cannot be absolutely sure if we can trust W&B.

It is probably an issue with the resolution of the graph. Zooming in to the values shows the correct result. Is it possible to alert the user in case this happens? Otherwise, important clues during the training of models might get obfuscated.

![1](https://user-images.githubusercontent.com/6742366/96273047-2ea2cc00-0fcf-11eb-8b5b-41c6d4e5e509.png)
![2](https://user-images.githubusercontent.com/6742366/96273050-2f3b6280-0fcf-11eb-8eb8-6ff149aecc84.png)

**Comment by vanpelt** [comment]
@habichta if you send us a link to the run in question we can debug the root cause.  My guess is you're actually logging NaN values where we render zeros in the chart.  You can see the raw data by using our public api to access all the data points: https://docs.wandb.com/library/api/examples#export-metrics-from-a-large-single-run-without-sampling

**Comment by ariG23498** [comment]
Hey @habichta 
Could you let us know if the issue is tackled with?
We are closing this issue due to inactivity of the thread. Please feel free to reopen this ticket if the issue persists. :smile:
